test/integration_testcase.go

```diff
@@ -118,7 +140,14 @@ func getFile(path string) (bool, string, error) {
 	if err != nil {
 		return false, "", nil
 	}
-	f, err := ioutil.ReadFile(path)
+
+	// Ensure it's the absolute, symlink-less path we're returning
```

Comment:
This comment does not seem to match with the actual goal of what's happening here - is it copied from elsewhere?